src/android/com/outsystems/firebase/analytics/FirebaseAnalyticsPlugin.java

```diff
@@ -163,6 +163,10 @@ private void setConsent(String consentSetting, CallbackContext callbackContext)
                 int typeValue = consentItem.getInt("Type");
                 int statusValue = consentItem.getInt("Status");
 
+                if (!consentItem.has("Type") || !consentItem.has("Status")) {
```

Comment:
We can probably do this validation before doing `consentItem.getInt("Type")` and `consentItem.getInt("Status")` right? Because in case the if statement is true, then we want to throw and don't need `typeValue` and `statusValue`